Automation failure : test_update_trend_positive

TestNames:
tests.foreman.ui.test_trend.TrendTest.test_update_trend_positive

Error Message
'NoneType' object has no attribute 'click'

Stacktrace
  File "/usr/lib64/python2.7/unittest/case.py", line 369, in run
    testMethod()
  File "/home/jenkins/workspace/satellite6-automation-downstream-rhel66-ui/tests/foreman/ui/test_trend.py", line 56, in test_update_trend_positive
    self.trend.update(name, 'clientcert', new_name)
  File "robottelo/ui/trend.py", line 45, in update
    edit_fld.click()
'\'NoneType\' object has no attribute \'click\'\n-

when executed locally using nosetests:
$ nosetests -c robottelo.properties -m test_update_trend_positive /home/tkolhar/git/robottelo/tests/foreman/ui/test_trend.py
ERROR: @Test: Update trend entity value
Traceback (most recent call last):
  File "/home/tkolhar/git/robottelo/robottelo/test.py", line 258, in tearDown
    self.take_screenshot(self._testMethodName)
  File "/home/tkolhar/git/robottelo/robottelo/test.py", line 246, in take_screenshot
    directory,
  File "/usr/lib64/python2.7/posixpath.py", line 77, in join
    elif path == '' or path.endswith('/'):
AttributeError: 'NoneType' object has no attribute 'endswith'


FAIL: @Test: Update trend entity value
Traceback (most recent call last):
  File "/home/tkolhar/git/robottelo/tests/foreman/ui/test_trend.py", line 54, in test_update_trend_positive
    self.assertIsNotNone(search)
AssertionError: unexpectedly None


XML: foreman-results.xml
Ran 1 test in 60.102s
FAILED (errors=1, failures=1)

it seems it had no object

--- a/tests/foreman/ui/test_trend.py
+++ b/tests/foreman/ui/test_trend.py
@@ -41,8 +41,8 @@ class TrendTest(UITestCase):
         @Assert: Trend entity is updated
 
         """
-        name = gen_string('utf8')
-        new_name = gen_string('utf8')
+        name = gen_string('alphanumeric')
+        new_name = gen_string('alphanumeric')
         with Session(self.browser) as session:
             make_trend(
                 session,

